src/constants.jl

```diff
@@ -1,22 +1,23 @@
+global const ETCD_VER = "v3.2.12"
```

Comment:
I'm kind of surprised that the tests even pass against the v3 API... didn't they deprecate the JSON API in favour of gRPC and protobuf?

Reply:
Yes. It still supports the v2 apis though. And `connect` specifies `v2` as the api to use.